skimage/__init__.py

```diff
@@ -62,12 +62,13 @@
 import warnings
 import sys
 
+from ._version import get_versions
```

Comment:
I'm not a fan of the `del`... maybe `from ._version import get_versions as _get_versions`?

Reply:
Ok. That was the boilerplate code from versioneer.